test/units/testsuite-46.sh

```diff
@@ -451,7 +451,7 @@ if command -v ssh &>/dev/null && command -v sshd &>/dev/null && ! [[ -v ASAN_OPT
         set +e
 
         systemctl is-active -q mysshserver.socket && systemctl stop mysshserver.socket
-        rm -f /tmp/homed.id_rsa /run/systemd/system/mysshserver{@.service,.socket}
```

Comment:
the commit msg is a bit misleading. it's not faster with ecdsa because the keys are smaller, but simply because EC algorithms are a million times faster than RSA